app/bundles/LeadBundle/Entity/Lead.php

```diff
@@ -228,6 +238,14 @@ public static function loadMetadata (ORM\ClassMetadata $metadata)
             ->cascadePersist()
             ->cascadeDetach()
             ->build();
+
+        $builder->createField('mobile', 'string')
```

Comment:
These two items are actually part of the SMS update. I'll remove them from this PR